config/initializers/0_clowder-config.rb

```diff
@@ -24,9 +23,9 @@
   redis_url = "#{config.inMemoryDb.hostname}:#{config.inMemoryDb.port}"
```

Comment:
Needs the schema. See https://github.com/RedHatInsights/compliance-backend/pull/905/files#diff-274fbb104eb7e6e64d378ecb4f260a28440db9fc32916c19337dae8aa46cf85aR21

Reply:
Just kidding! Do not make this change please :laughing:

Reply:
Sorry, this change was for `rbac_url` not `redis_url`. Redis uses its own `redis://` schema. Anyway, it's fixed in https://github.com/RedHatInsights/compliance-backend/pull/905